Documentation/ApiOverview/CropVariants/General/Index.rst

```diff
@@ -0,0 +1,139 @@
+.. include:: ../../Includes.txt
+
+.. _cropvariants:
+
+=====================
+General Configuration
+=====================
+
+The following examples are basically meant to add one single
+cropping configuration to sys_file_reference,  which will then apply in every
+record, which reference images.
+
+In this example we configure two crop variants, one with the id "mobile",
+one with the id "desktop". The array key defines the crop variant id, which will be used
```

Comment:
that would be the keys in allowedAspectRatios i.e. 4:3 and NaN?

Reply:
yep, but maybe these keys are not relevant at all. 
They only define the preconfigured aspect ratios in backend UI, I think